clients/cli/src/copy.rs

```diff
@@ -1,64 +1,64 @@
+use std::cell::Cell;
+use std::io::Write;
 use std::path::PathBuf;
 
+use lockbook_core::service::import_export_service::ImportStatus;
 use lockbook_core::{
-    create_file_at_path, get_file_by_path, import_file, CreateFileAtPathError, Error as CoreError,
-    Error, GetFileByPathError, ImportFileError,
+    create_file_at_path, get_file_by_path, import_files, CoreError, CreateFileAtPathError,
+    GetFileByPathError, ImportFileError,
 };
+use lockbook_models::file_metadata::DecryptedFileMetadata;
 
 use crate::error::CliResult;
 use crate::utils::{account, config};
 use crate::{err, err_unexpected};
-use lockbook_core::service::import_export_service::ImportExportFileInfo;
-use lockbook_models::file_metadata::DecryptedFileMetadata;
 
 pub fn copy(disk_paths: &[PathBuf], lb_path: &str) -> CliResult<()> {
     account()?;
 
-    let file_metadata = get_or_create_file(lb_path)?;
-
-    let import_progress = |info: ImportExportFileInfo| {
-        println!("importing: {} to {}", info.disk_path.display(), info.lockbook_path);
+    let total = Cell::new(0);
```

Comment:
Is `Cell` actually needed here? These vars can just be local to the closure right?

Reply:
Nope it is actually needed here

Reply:
nvm